knative-operator/pkg/controller/knativeserving/servicemesh/servicemesh.go

```diff
@@ -239,6 +242,39 @@ func isServiceMeshMemberRollReady(servingNamespace string, api client.Client) (b
 	return false, nil
 }
 
+// create wide-open networkpolicies for the service mesh components
+func installNetworkPolicies(instanceNamespace string, api client.Client) error {
+	log.Info("Installing Mesh Network Policies")
+	namespace := ingressNamespace(instanceNamespace)
+	const path = "deploy/resources/serviceMesh/networkpolicies.yaml"
+
+	manifest, err := mf.NewManifest(path, false, api)
+	if err != nil {
+		log.Error(err, "Unable to create Mesh Network Policy install manifest")
+		return err
+	}
+	smcp := &maistrav1.ServiceMeshControlPlane{}
+	ready, err := isServiceMeshControlPlaneReady(instanceNamespace, api)
+	if err != nil {
+		return err
+	}
+	if ready {
+		if err := api.Get(context.TODO(), client.ObjectKey{Namespace: namespace, Name: smcpName}, smcp); err != nil {
+			return err
+		}
+	}
+	transforms := []mf.Transformer{mf.InjectOwner(smcp), mf.InjectNamespace(namespace)}
```

Comment:
ownerreferences for smcp causes some minor issue. For example, this network-policy remains if we uninstall serverless-operator. _But_ I guess this should be align with Maistra's istio-ingressgateway. So, let's leave this as you implemented.

Reply:
The GC should delete the NP when the SMCP is deleted due to the ownerref. And the KS controller will delete the entire `knative-serving-ingress` namespace when the KS CR is deleted. So I think everything will be properly cleaned up. Of course, none of that happens if this operator isn't running (i.e. uninstall serverless-operator), but that's expected, right?